homeassistant/components/pandora/media_player.py

```diff
@@ -310,7 +310,7 @@ def _update_stations(self) -> None:
         station_lines = self._pianobar.before or ""
         _LOGGER.debug("Getting stations: %s", station_lines)
         self._attr_source_list = []
-        for line in station_lines.split("\r\n"):
+        for line in station_lines.splitlines():
```

Comment:
Small cleanup. `splitlines` works better in that case. From the Python docs:

> Unlike [split()](https://docs.python.org/3/library/stdtypes.html#str.split) when a delimiter string sep is given, this method returns an empty list for the empty string, and a terminal line break does not result in an extra line:
>
> ```py
> >>> "".splitlines()
> []
> >>> "One line\n".splitlines()
> ['One line']
> ```
>
> For comparison, split('\n') gives:
>
> ```py
> >>> ''.split('\n')
> ['']
> >>> 'Two lines\n'.split('\n')
> ['Two lines', '']
> ```

https://docs.python.org/3/library/stdtypes.html#str.splitlines